auction-app-frontend/vite.config.js

```diff
@@ -1,12 +1,16 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
   resolve: {
     alias: {
-      src: '/src',
+      src: "/src",
+      "src/scss": "/src/scss",
```

Comment:
Since we are using only imports in this format `src/...` you can remove configuration for others.